project/Dependencies.scala

```diff
@@ -195,6 +197,8 @@ object Dependencies {
 
   val actorTestkitTyped = l ++= Seq(Provided.logback, Provided.junit, Provided.scalatest, Test.scalatestJUnit)
 
+  val pki = l ++= Seq(asnOne, Test.scalatest)
```

Comment:
This fixes the unidoc issue for me:

```suggestion
  val pki = l ++=
      Seq(
        asnOne,
        // pull up slf4j version from the one provided transitively in asnOne to fix unidoc
        Compile.slf4jApi % "provided",
        Test.scalatest)
```

Reply:
thanks @jrudolph